crates/rnote-ui/src/appwindow/actions.rs

```diff
@@ -1171,6 +1186,12 @@ impl RnAppWindow {
         app.set_accels_for_action("win.pen-style::eraser", &["<Ctrl>4", "<Ctrl>KP_4"]);
         app.set_accels_for_action("win.pen-style::selector", &["<Ctrl>5", "<Ctrl>KP_5"]);
         app.set_accels_for_action("win.pen-style::tools", &["<Ctrl>6", "<Ctrl>KP_6"]);
+        (1..=9).for_each(|i| {
+            app.set_accels_for_action(
+                &format!("win.set-color-{i}"),
+                &[&format!("<Ctrl><Shift>{i}"), &format!("<Alt>KP_{i}")],
```

Comment:
The keypad shortcuts still use <kbd>Alt</kbd> atm.
```suggestion
                &[&format!("<Ctrl><Shift>{i}"), &format!("<Ctrl><Shift>KP_{i}")],
```